### PR TITLE
Add "Contao"

### PR DESCRIPTION
**Pull request by zonky2**
Add "Contao"

### CODE REVIEW

File: languages/en/default.xlf [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <file datatype="php" date="2018-10-22T10:57:28+02:00" original="default" source-language="en">
     <body>
       <trans-unit id="MSC.notification_center.warn_legacy_smtp">
-        <source>[Notification Center]: Using custom SMTP settings has been changed in Contao 4.10. Please visit &lt;a href=&quot;https://docs.contao.org/manual/en/system/settings/#different-e-mail-configurations-and-sender-addresses&quot;&gt;the documentation&lt;/a&gt; to learn how to define SMTP settings. Then, please update all email gateways that need this feature and disable "Override SMTP settings".</source>
+        <source>[Notification Center]: Using custom SMTP settings has been changed in Contao 4.10. Please visit &lt;a href=&quot;https://docs.contao.org/manual/en/system/settings/#different-e-mail-configurations-and-sender-addresses&quot;&gt;the Contao documentation&lt;/a&gt; to learn how to define SMTP settings. Then, please update all email gateways that need this feature and disable "Override SMTP settings".</source>
```
Comment (fritzmg) [inline languages/en/default.xlf:6]:
```suggestion
        <source>[Notification Center]: Using custom SMTP settings has been changed in Contao 4.10. Please visit &lt;a href=&quot;https://docs.contao.org/manual/en/system/settings/#different-e-mail-configurations-and-sender-addresses&quot; style=&quot;text-decoration:underline&quot;&gt;the Contao documentation&lt;/a&gt; to learn how to define SMTP settings. Then, please update all email gateways that need this feature and disable "Override SMTP settings".</source>
```

Comment (zonky2) [reply languages/en/default.xlf:6]: maybe it´s better to backport this https://github.com/contao/contao/blob/57ea584579919a096ec0ff5b43246c5bbfa3b98f/core-bundle/contao/themes/flexible/main.css#L580-L582 to C4.13

Comment (fritzmg) [reply languages/en/default.xlf:6]: Forcing the underline does not hurt.

**Comment by zonky2** [comment]
one year later...

**Pull request closed without merging**